package.json

```diff
@@ -62,27 +62,26 @@
   },
   "devDependencies": {
     "@nextcloud/upload": "^1.1.0-beta.1",
-    "@nextcloud/webpack-vue-config": "^5.3.0",
-    "@rollup/plugin-commonjs": "^23.0.2",
+    "@nextcloud/webpack-vue-config": "^5.4.0",
+    "@rollup/plugin-commonjs": "^23.0.3",
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@rollup/plugin-typescript": "^10.0.1",
     "@types/cypress": "^1.1.3",
     "@types/dockerode": "^3.3.12",
     "@types/jest": "^29.2.2",
     "@vue/cli-service": "^5.0.8",
     "@vue/tsconfig": "^0.1.3",
-    "cypress": "^12.0.1",
```

Comment:
we lost cypress? :grin:

Reply:
This is from when I updated cypress in this PR. Node / NPM starting with NPM 7 installs all peer dependencies automatically when installing a package (e.g. for developing / testing).
So all dependencies are only required once in either dependencies, devDependencies or peerDependencies.

But if you like keeping the dependency twice to make clear it is also a devDependency I revert this :smile:

Reply:
Edit: I reverted this as this is unrelated to this PR so do not mix up topics here :sweat_smile:

Reply:
> But if you like keeping the dependency twice to make clear it is also a devDependency I revert this

Yes, we do this. peerDependencies AND devDependencies is required. They serve different purpose